tbans/tbans.yaml

```diff
@@ -0,0 +1,17 @@
+service: tbans
+
+runtime: python27
+api_version: 1
+threadsafe: true
+
+builtins:
+- appstats: on
+- deferred: on
+
+handlers:
+- url: .*
+  script: tbans_service.app
+  # login: admin
```

Comment:
This needs to be fixed - I keep getting 302's when hitting this from the mobile API